docs/authorization.md

```diff
@@ -0,0 +1,94 @@
+<!--
+---
+linkTitle: "Authorization"
+weight: 1
+---
+-->
+
+# Authorization
+
+This document explains use of Kubernetes "Rules Based Access Control" 
+or ["RBAC"](https://kubernetes.io/docs/reference/access-authn-authz/authorization/), beyond the 
+definition of various RBAC objects in the `config` folder of this project.
+
+## Controller Access to ClusterTasks from PipelineRuns and TaskRuns
+
+By default, Tekton allows any `PipelineRun` or `TaskRun` to access any ClusterTask because the 
+controller(s) has access to any ClusterTask.  However, if you happened to log into the cluster
+using the credentials of the `ServiceAccount` associated with the `PipelineRun` or `TaskRun` and ran
+
+```bash
+kubectl  auth can-i get clustertask <cluster task name>
+``` 
+
+you will get `false` because by default the `ServiceAccounts` used in `PipelineRuns` and `TaskRuns` are
+not granted permission to access the cluster scoped resource `ClusterTask`.  And remember, if no `ServiceAccount` is 
+set on the `PipelineRun` or `TaskRun`, the `default` `ServiceAccount` is used by Kubernetes when running the underlying
+`Pod`.
+
+If as an administrator, you want to control which `PipelineRun` and `TaskRun` instances, via their `ServiceAccounts`, 
+have access to given `ClusterTasks`, consider these steps:
+
+- First, as a convenience, Tekton adds viewing of `ClusterTask` as part of its aggregation of permissions to the default
+`view` / `edit` / `admin` `ClusterRoles` present with any Kubernetes cluster.  You can then assign one of those default
+`ClusterRoles` to any `ServiceAccount` that should have access to `ClusterTasks`
+- If one of those default `ClusterRoles` is too broad, then create more precise `Role` / `RoleBinding` or `ClusterRole` 
+/ `ClusterRoleBinding` and assign those to the `ServiceAccounts` in questions.
+- Then, an update to the Tekton webhook `Deployment` named `tekton-pipelines-webhook` in the `tekton-pipelines` namespace 
+is needed to enable the validating admission webhook that will ensure that the `ServiceAccount` associated with a 
+`PipelineRun` or `TaskRun` can read its referenced `ClusterTask`.
+- The specific update needed is setting the environment variable `ENABLE_CLUSTER_TASK_ACCESS_VALIDATION` to "true" on
+the webhook's underlying `Container`.
+- Once set, enforcement that the necessary permissions exist for `ClusterTask` access should start. 
+
+## Pod Security Policy
```

Comment:
I went ahead and added the Pod Security Policy best practice suff we discussed in the API WG, since I was creating this new doc file where the content would live.

It certainly is a separate concept from ClusterTask access.

If desired I'll move this to separate PR.